app/js/views/app_view.js

```diff
@@ -1,16 +1,35 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'text!templates/app_view.ejs',
+  'collections/todo'
+], function ($, Backbone, indexTemplate, todoCollection) {
   return Backbone.View.extend({
     el: $('#todo-app'),
+    input: $('#input'),
 
-    currentDate: function () {
-      return new Date();
+    events: {
+      'click #addButton': 'addItems',
+      'keydown #input': 'keyAction'
     },
 
+    addItems: function(){
+      var newItem = input.value;
+      if(_.trim(newItem) !== ""){
+        this.collection.add({name: newItem, done: false});
+      }
+      input.value = "";
+    },
+
+    keyAction: function(e){
+      if(e.which === 13){
```

Comment:
constants can be extracted to a variable to reflect what they actually mean like `var ENTER_KEY_CODE = 13`